holoviews/core/dimension.py

```diff
@@ -780,7 +780,9 @@ def __setstate__(self, d):
         except:
             self.warning("Could not unpickle custom style information.")
         self.__dict__.update(d)
-
+        # TODO: super's setstate not called?
```

Comment:
Seems like an oversight but I don't think the super would do anything else in this case unless I'm mistaken.

Reply:
Okay, thanks for explaining that. I have no objection to merging this without a super call due to the ``setattr`` issue. I can see how that would break on Overlay and Layout objects.